docs/working-with-projects/working-with-images/create-tag-retention-rules.md

```diff
@@ -15,58 +15,68 @@ A tag retention rule has 3 filters that are applied sequentially, as described i
+In this example the rule retains the 10 most recently pulled tagged artifacts in repository A, and the 6 of the artifacts in each of the 4 repositories B to E. So, a total of 34 artifacts are retained in the project. 
+In other words, the rule does not treat all of the artifacts in repositories A to E as a single pool from which to choose the 10 most recent artifacts. 
+So, even if the 11th to 100th tags in repository A have been pulled more recently than any of the tags in repositories B to E, all of the tagged artifacts in repositories B to E are retained, because each of those repositories has fewer than 10 tags, and all untagged artifacts are deleted.
 
 ### Example 2
 
 This example uses the same project and repositories as example 1, but sets the retention policy to retain the images in each repository that have been pulled in the last 7 days.
 
-In this case, all of the images in repository A are retained because they have been pulled in the last 7 days. None of the images in repositories B to E are retained, because none of them has been pulled in the last week. In this example, 100 images are retained, as opposed to 34 images in example 1.
+In this case, all of the 100 tagged artifacts in repository A are retained because they have been pulled in the last 7 days. None of the images in repositories B to E are retained, because none of them has been pulled in the last week. In this example, 100 images are retained, as opposed to 34 images in example 1.
+And all untagged artifacts are deleted.
+
+### Example 3
+
+This example uses the same project and repositories as example 2, but checked "untagged artifacts", then all artifacts in repository A are retained.
+
+In this case, all of the 103 artifacts in repository A are retained because they have been pulled in the last 7 days. None of the images in repositories B to E are retained, because none of them has been pulled in the last week. In this example, 103 artifacts are retained, as opposed to 100 artifacts in example 2.
+And all untagged artifacts are deleted.
 
 ### Tag Retention Rules and Native Docker Tag Deletion
 
-**WARNING**: Due to native Docker tag deletion behavior, there is an issue with the current retention policy implementation. If you have multiple tags that refer to the same SHA digest, and if a subset of these tags are marked for deletion by a configured retention policy, all of the remaining tags would also be deleted. This violates the retention policy, so in this case all of the tags are retained. This issue will be addressed in a future update release, so that tag retention policies can delete tags without deleting the digest and other shared tags.
+**Note**: If an artifact has several tags, and one tag is matched retain policy, the artifact and all its tags will be retained. 
```

Comment:
-> 'and only a partial set of tags are matched via the retention policy, then the artifact and all its tags will be retained.  In other words, retention is matched at the tag level but retention / deletion is carried out at the artifact level, with retention fully preserving the artifact including all of its tags'